test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14647,6 +14647,11 @@ def __init__(self, job, cmd, host):
         self.cmd = cmd
         self.jobid = None
         self.hostname = host
+        self.runas_user = ""
+        if self.du.get_platform() == "shasta":
+            self.runas_user = PbsUser.get_user(job.username)
+            self.hostname = self.runas_user.host \
+                if self.runas_user.host else self.hostname
```

Comment:
would it be simpler if we just have -

if self.runas_user.host

    self.hostname = self.runas_user.host